client/extensions/wp-super-cache/components/debug/index.jsx

```diff
@@ -0,0 +1,201 @@
+import moment from 'moment';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import FormTextInput from 'components/forms/form-text-input';
+import FormToggle from 'components/forms/form-toggle/compact';
+import SectionHeader from 'components/section-header';
+import WrapSettingsForm from '../wrap-settings-form';
+
+class DebugTab extends Component {
+	static propTypes = {
+		fields: PropTypes.object,
+		translate: PropTypes.func.isRequired,
+	};
+
+	static defaultProps = {
+		fields: {},
+	};
+
+	render() {
+		const {
+			fields: {
+				wp_super_cache_debug,
+				wp_cache_debug_ip,
+				wp_super_cache_comments,
+				wp_super_cache_front_page_check,
+				wp_super_cache_front_page_clear,
+				wp_super_cache_front_page_text,
+				wp_super_cache_front_page_notification,
+			},
+			handleAutosavingToggle,
+			handleChange,
+			handleSubmitForm,
+			isRequesting,
+			isSaving,
+			translate,
+		} = this.props;
+
+		return (
+			<div>
+				<SectionHeader label={ translate( 'Debug' ) }>
+					<Button
+						compact
+						primary
+						disabled={ isRequesting || isSaving }
+						onClick={ handleSubmitForm }>
+						{ isSaving
+							? translate( 'Saving…' )
+							: translate( 'Save Settings' )
+						}
+					</Button>
+				</SectionHeader>
+				<Card>
+					<form>
+						<FormFieldset>
+							<FormToggle
+								checked={ !! wp_super_cache_debug }
+								disabled={ isRequesting || isSaving }
+								onChange={ handleAutosavingToggle( 'wp_super_cache_debug' ) }>
+								<span>
+									{ translate( 'Enable Debugging' ) }
+								</span>
+							</FormToggle>
+						</FormFieldset>
+						<div className="wp-super-cache__debug-fieldsets">
+							<FormFieldset>
+								<FormLabel htmlFor="ipAddress">
+									{ translate( 'IP Address' ) }
+								</FormLabel>
+								<FormTextInput
+									disabled={ isRequesting || isSaving || ! wp_super_cache_debug }
+									id="ipAddress"
+									onChange={ handleChange( 'wp_cache_debug_ip' ) }
+									value={ wp_cache_debug_ip || '' } />
+								<FormSettingExplanation>
+									{ translate(
+										'(only log requests from this IP address)',
+									) }
+								</FormSettingExplanation>
+							</FormFieldset>
+							<FormFieldset>
+								<FormToggle
+									checked={ !! wp_super_cache_comments }
+									disabled={ isRequesting || isSaving || ! wp_super_cache_debug }
+									onChange={ handleAutosavingToggle( 'wp_super_cache_comments' ) }>
+									<span>
+										{ translate( 'Cache Status Messages' ) }
+									</span>
+								</FormToggle>
+								<FormSettingExplanation>
+									{
+										translate(
+											'Display comments at the end of every page like this:' +
+											'{{pre}}' +
```

Comment:
I'm seeing this error in console:

![console-error](https://user-images.githubusercontent.com/1190420/27915622-e5802afe-6234-11e7-989b-84576d8afbbe.jpg)

There's a code block in the _Lock Down_ section on the _Advanced_ tab. Perhaps we could do something similar here.